README.md

```diff
@@ -1,22 +1,37 @@
-# Turborepo starter
+# EventSea: A Decentralized Events Platform
 
-This is an official starter Turborepo.
+---
 
-## Using this example
+### Overview:
 
-Run the following command:
+EventSea is a decentralized application (DApp) designed to revolutionize the way events are organized and attended. Built on the top of Linea, EventSea offers a transparent, secure, and user-centric platform for event creation and ticketing.
 
-```sh
-npx create-turbo@latest
-```
+![image](./eventsea.png)
+
+#### Key Features:
+
+- Decentralized Event Creation:
+  Users have the freedom to create their own events without the need for intermediaries or centralized platforms.
+  Complete control over event details, ticketing, and updates.
+
+- Mintable Tickets:
+  Participants can mint their own tickets, ensuring authenticity and reducing the risk of counterfeit tickets.
+  Each ticket is a unique digital asset on the blockchain, providing traceability and security.
 
-## What's inside?
```

Comment:
we don't have this yet. 
may be add it to a road map or something ?

Reply:
Okay I will do that. @chin-flags